CHANGELOG.md

```diff
@@ -3580,6 +3580,8 @@ If you're still not using React 15 you can keep the old behavior by setting the
 ### Added
  * First revision
 
+<!-- common links -->
```

Comment:
this does _work_, but it'd be nice to not rely on it, and instead find a way to locate all referenced links